src/reactor.js

```diff
@@ -27,6 +28,7 @@ class Reactor {
     const baseOptions = debug ? DEBUG_OPTIONS : PROD_OPTIONS
     const initialReactorState = new ReactorState({
       debug: debug,
+      cache: config.cache || DefaultCache(),
```

Comment:
I'm thinking probably exposing both the DefaultCache and LRUCache on the `Nuclear` object, and allowing one to be passed into creating a new reactor.